src/utils/encoders.ts

```diff
@@ -71,3 +122,37 @@ export const decodeOrder = (order: EncodedOrder): DecodedOrder => {
     ).toString(),
   };
 };
+
+/**
+ * Use the ratio between the z and the price geometric average to calculate what z
+ * value should be used for the other order in order to preserve correlation between
+ * the two orders - and provide the liquidity that should be used to achieve that.
+ */
```

Comment:
Price geometric mean is irrelevant here.
You can write something like:
> Use the capacity of the other order along with the prices of this order, in order to calculate the capacity that this order needs to have in order for its marginal price to be set according to the given input

Reply:
thanks